lldb/source/Target/SwiftLanguageRuntime.cpp

```diff
@@ -2095,25 +2095,31 @@ void SwiftLanguageRuntime::DidFinishExecutingUserExpression(
 
 bool SwiftLanguageRuntime::IsABIStable() { FORWARD(IsABIStable); }
 
-struct AsyncUnwindRegisters {
+// The target specific register numbers (eh_frame/dwarf) used for async
+// unwinding.
```

Comment:
Annoying nitpicking: using `///` here will format more nicely in some IDEs, even if it won't make it into the doxygen html because it's local.

Reply:
done